Validation/RecoHI/python/muonValidationHeavyIons_cff.py

```diff
@@ -68,26 +69,26 @@
 MTVhi.maxPt = cms.double(100)
 
 # MuonTrackValidator parameters
-trkMuonTrackVMuonAssocHI = MTVhi.clone()
-trkMuonTrackVMuonAssocHI.associatormap  = 'tpToTkMuonAssociationHI'
-trkMuonTrackVMuonAssocHI.label          = ['cutsRecoTrkMuons']
-trkMuonTrackVMuonAssocHI.muonHistoParameters = trkMuonHistoParameters
-
-glbMuonTrackVMuonAssocHI = MTVhi.clone()
-glbMuonTrackVMuonAssocHI.associatormap = 'tpToGlbMuonAssociationHI'
-glbMuonTrackVMuonAssocHI.label           = ['globalMuons']
-glbMuonTrackVMuonAssocHI.muonHistoParameters = glbMuonHistoParameters
-
-staMuonTrackVMuonAssocHI = MTVhi.clone()
-staMuonTrackVMuonAssocHI.associatormap = 'tpToStaMuonAssociationHI'
-staMuonTrackVMuonAssocHI.label = ('standAloneMuons',)
-staMuonTrackVMuonAssocHI.muonHistoParameters = staMuonHistoParameters
-
-staUpdMuonTrackVMuonAssocHI = MTVhi.clone()
-staUpdMuonTrackVMuonAssocHI.associatormap = 'tpToStaUpdMuonAssociationHI'
-staUpdMuonTrackVMuonAssocHI.label = ('standAloneMuons:UpdatedAtVtx',)
-staUpdMuonTrackVMuonAssocHI.muonHistoParameters = staUpdMuonHistoParameters
-
+trkMuonTrackVMuonAssocHI = MTVhi.clone(
+associatormap  = 'tpToTkMuonAssociationHI',
+label          = ('cutsRecoTrkMuons'),
+muonHistoParameters = trkMuonHistoParameters
+)
```

Comment:
I think that the squared parentheses are needed to signify that these are vectors of InputTags: perhaps for mono-dimensional vectors it doesn't matter, but I would let it indicated as well.

At the same time, in order to improve the readability, could you please indent all these cloned configs (here, above and below), for example:

```suggestion
trkMuonTrackVMuonAssocHI = MTVhi.clone(
   associatormap  = 'tpToTkMuonAssociationHI',
   label          = ['cutsRecoTrkMuons'],
   muonHistoParameters = trkMuonHistoParameters
)
```

Reply:
@perrotta  Okay. let me follow your comments. Sorry, I have missed replying to this.